src/plugins/themes.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const fs = require("fs");
+const Helper = require("../helper");
+const colors = require("colors/safe");
+const path = require("path");
+const themes = new Map();
+
+module.exports = {
+	get: get,
+	fileName: fileName
+};
+
+fs.readdir("client/themes/", (err, builtInThemes) => {
+	if (err) {
+		return;
+	}
+	builtInThemes
+		.filter((theme) => theme.endsWith(".css"))
+		.map(makeLocalThemeObject)
+		.forEach((theme) => themes.set(theme.name, theme));
+});
+
+fs.readdir(path.join(Helper.getPackagesPath(), "node_modules"), (err, packages) => {
+	if (err) {
+		return;
+	}
+	packages
+		.map(makePackageThemeObject)
+		.forEach((theme) => {
+			if (theme) {
+				themes.set(theme.name, theme);
+			}
+		});
+});
+
+function get() {
+	return Array.from(themes.values());
+}
+
+function fileName(module) {
```

Comment:
I see a mix of `filename` and `fileName` around here, could we stick with `filename` only?

Reply:
Will. I am never sure whether filename is 1 or 2 words therefore what it should be in camel case.

Reply:
Replacing all to filename.